interface/patient_file/summary/create_portallogin.php

```diff
@@ -12,21 +12,20 @@
  * @author    Tyler Wrenn <tyler@tylerwrenn.com>
  * @author    Stephen Nielson <snielson@discoverandchange.com>
  * @author    Stephen Waite <stephen.waite@open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
  * @copyright Copyright (c) 2011 Z&H Consultancy Services Private Limited <sam@zhservices.com>
  * @copyright Copyright (c) 2018-2019 Brady Miller <brady.g.miller@gmail.com>
  * @copyright Copyright (c) 2020 Tyler Wrenn <tyler@tylerwrenn.com>
  * @copyright Copyright (c) 2022 Discover and Change, Inc <snielson@discoverandchange.com>
  * @copyright Copyright (c) 2022 Stephen Waite <stephen.waite@open-emr.org
+ * @copyright Copyright (c) 2017-2023 Jerry Padgett <sjpadgett@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 require_once("../../globals.php");
 require_once('../../../library/amc.php');
 
-use OpenEMR\Common\ {
-    Csrf\CsrfUtils,
-    Utils\RandomGenUtils,
-};
+use OpenEMR\Common\{Csrf\CsrfUtils,};
```

Comment:
was this meant to remove Utils\RandomGenUtils ?

Reply:
not used so not needed

Reply:
sounds good, then would change to since no longer compounding:
```
use OpenEMR\Common\Csrf\CsrfUtils;
```

Reply:
btw already is

Reply:
sounds like maybe issue with PR since I see `use OpenEMR\Common\{Csrf\CsrfUtils,};` instead of `use OpenEMR\Common\Csrf\CsrfUtils;` in line above